### PR TITLE
translate pattern chain-of-responsability to pt-br

### PR DESCRIPTION
**Pull request by ricardo-ramos-moura**
Translate pattern chain-of-responsability to pt-br (Brazilian portuguese)

### CODE REVIEW

**Review by V1nic1us — APPROVED** [review]
LGTM

File: localization/pt/chain-of-responsability/README.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
+---
+title: Chain of responsibility
+category: Behavioral
+language: en
```
Comment (iluwatar) [inline localization/pt/chain-of-responsability/README.md:4]: Here should be the language code. See https://github.com/iluwatar/java-design-patterns/wiki/15.-Support-for-multiple-languages

**Review by iluwatar — CHANGES_REQUESTED** [review]

File: localization/pt/chain-of-responsability/README.md (hunk 1 above)
Comment (ricardo-ramos-moura) [reply localization/pt/chain-of-responsability/README.md]: @iluwatar updated language pt in README.md

**Review by iluwatar — APPROVED** [review]

**Comment by iluwatar** [comment]
Looks good! Thank you for the contribution 🎉

@all-contributors please add @ricardo-ramos-moura for translation

**Pull request merged**